docs/01-engage/02-accounts/02-creating-accounts/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Creating Accounts
+sidebar_position: 2
+---
+
+Accounts in **ENGAGE** are separate individuals (donors or supporters) and organizations that donate charity. This article will walk you through creating a new account via ENGAGE.
```

Comment:
This doesn't read well.

Donors and supporters are mostly synonyms.

You also don't want to say via Engage when I think you mean within Engage.

A couple of things I think are missed:

Needs a more general introduction, something along the lines of:

An account in Engage represents an individual or organisation (business, school, community group etc.) which interacts with your organisation and where you wish to keep track of their contact details, preferences, donations, sponsorships, etc. in a single place.

In Engage accounts are either individual or organisations. They are separate as in most locations the rules around tax relief, data protection and many other areas are different between the two.

We also advise against creating a household account for a couple or family. In most locations tax relief and consent options are specific to an individual and will not work as expected if individual accounts are combined. Instead, accounts can be grouped into households after exporting if you wish to do so for purposes such as a postal mailing.

Also should add a passing comment at the top that to avoid duplicates and the problems those cause, they should first search for an account with link to that screen.